### PR TITLE
COMMERCE-9302 Integration test implementation for the CartResourceTest class

### PR DESCRIPTION
**Pull request by ghost**
https://issues.liferay.com/browse/COMMERCE-9302

### CODE REVIEW

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Review by luca-pellizzon — CHANGES_REQUESTED** [review]
Just some clarifications, thanks

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/cart/resource/v1_0/test/CartResourceTest.java [hunk 1]
```diff
@@ -15,14 +15,232 @@
...
+	private CommerceOrder _getCommerceOrder() throws Exception {
+		_commerceOrder = _commerceOrderLocalService.addCommerceOrder(
+			_user.getUserId(), _commerceChannel.getGroupId(),
+			_commerceAccount.getCommerceAccountId(),
+			_commerceCurrency.getCommerceCurrencyId(), 0);
+
+		return _commerceOrder;
+	}
+
+	private CommerceAccount _commerceAccount;
```
[the remark this answers is not in the document]
Comment (CrescenzoRegaLF) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/cart/resource/v1_0/test/CartResourceTest.java:225]: on CommerceAccount the annotation @DeleteAfterTestRun gives errors

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/cart/resource/v1_0/test/CartResourceTest.java [hunk 2]
```diff
@@ -15,14 +15,232 @@
...
+		_commerceChannel = CommerceTestUtil.addCommerceChannel(
+			testGroup.getGroupId(), _commerceCurrency.getCode());
+	}
+
+	@After
+	@Override
+	public void tearDown() throws Exception {
+		super.tearDown();
+
+		if (_commerceOrder != null) {
```
[the remark this answers is not in the document]
Comment (CrescenzoRegaLF) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/cart/resource/v1_0/test/CartResourceTest.java:80]: yes we can use the @DeleteAfterTestRun annotation

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:forward

**Pull request closed without merging**